crypto/crypto.go

```diff
@@ -17,6 +18,7 @@ const (
 
 // TODO - encryption mode consts
 
+// Encrypted represents a ciphertext with related inputs
 type Encrypted struct {
 	Ciphertext string
 	Mode       string
```

Comment:
Agreed, although I think it was like that since my RSA stuff.
On 26 Mar 2015 19:26, "Jon Bonazza" <notifications@github.com> wrote:

> In crypto/crypto.go
> <https://github.com/pki-io/core/pull/59#discussion_r27247131>:
>
> > @@ -17,6 +18,7 @@ const (
> >
> >  // TODO - encryption mode consts
> >
> > +// Encrypted represents a ciphertext with related inputs
> >  type Encrypted struct {
> >  	Ciphertext string
> >  	Mode       string
>
> This is my fault, but we should probably be explicit with the
> Encrypted.Mode field and make its type Mode instead of string.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pki-io/core/pull/59/files#r27247131>.
>